src/main/java/org/teamresistance/frc/Drive.java

```diff
@@ -0,0 +1,38 @@
+package org.teamresistance.frc;
+
+import org.strongback.components.Stoppable;
+import org.strongback.components.ui.ContinuousRange;
+import org.strongback.drive.MecanumDrive;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Drive implements Stoppable {
+
+  private final MecanumDrive robotDrive;
+  private final ContinuousRange translateXSpeed;
+  private final ContinuousRange translateYSpeed;
+  private final ContinuousRange rotateSpeed;
+
+  public Drive(MecanumDrive robotDrive, ContinuousRange translateXSpeed, ContinuousRange translateYSpeed, ContinuousRange rotateSpeed) {
```

Comment:
Probably rename the speeds to just `xSpeed` and `ySpeed`. I believe it'll be understood to mean translation, especially with this being the drive class. Also, good job with assigning fields.